### PR TITLE
ModernGroceryBanner-All

### PR DESCRIPTION
**Pull request by m-wilk**
Please about feedback

### CODE REVIEW

File: app/src/assets/scss/custom/ui/organism/modern-grocery-banner/modern-grocery-banner.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.es-modern-grocery-banner-inner-bg {
+    background: url("../../../../../img/grocery-bg-2.png") no-repeat center center;
+    background-size: cover;
+}
```
Comment (andust) [inline app/src/assets/scss/custom/ui/organism/modern-grocery-banner/modern-grocery-banner.scss:4]: please prettier

File: app/src/ui/organism/modern-grocery-banner/ModernGroceryBanner.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+const ModernGroceryBanner = ({background}) => {
+    return(
+        <div className={`pt-5 pb-5 ${background}`}>
+             <div className="text-center container es-modern-grocery-banner-inner-bg pt-5">
+                <h2 className="es-h-33 league-spartan-bold text-white m-0">Get the best deal for Modern Grocery</h2>
+                <CustomLink className="bg-warning mt-4 mb-5" isArrow={true}>Shop Now</CustomLink>
+             </div>
+        </div>
+       
+    )
```
Comment (andust) [inline app/src/ui/organism/modern-grocery-banner/ModernGroceryBanner.js:12]: please prettier :P

**Review by andust — APPROVED** [review]

File: app/src/assets/scss/custom/ui/organism/modern-grocery-banner/modern-grocery-banner.scss (hunk 1 above)
Comment (m-wilk) [reply app/src/assets/scss/custom/ui/organism/modern-grocery-banner/modern-grocery-banner.scss:4]: ok

File: app/src/ui/organism/modern-grocery-banner/ModernGroceryBanner.js (hunk 2 above)
Comment (m-wilk) [reply app/src/ui/organism/modern-grocery-banner/ModernGroceryBanner.js:12]: ok

**Pull request merged**